### ISSUE TITLE
Exclude $v in formation of series hyperlink

### ISSUE THREAD

**Issue by seanluyk** [ualbertalib/discovery#1542]
**Describe the bug**
Clicking on a series link currently searches both the series and the volume number

**To Reproduce**
1. Go to https://search.library.ualberta.ca/catalog/670890
2. Click on series https://search.library.ualberta.ca/catalog?search_field=advanced&title=%22Theory+and+history+of+literature.+v.+22-23%22
3. Note [URL](https://search.library.ualberta.ca/catalog?search_field=advanced&title=%22Theory+and+history+of+literature.+v.+22-23%22) and that only 1 result is returned because volume is searched

**Expected behavior**
Clicking on series searches just the 830$a as a string

**Additional context**
Referenced in various other issues #1101 #941. Please verify that this is correct with @thelinkresolver

**Comment by ConnorSheremeta** [comment]
@seanluyk is it fine that the 490v record is removed completely from the 'Series' field? (See screenshot [here](https://github.com/ualbertalib/discovery/pull/1606/). If not, I'm not sure how this would be done as core blacklight creates the link based off of the whole field.

**Comment by seanluyk** [comment]
@ConnorSheremeta @weiweishi just saw the PR for this and unfortunately, I think $v needs to still be displayed, and given this limitation in core Blacklight, I'm not sure how we solve this? Stanford seems to be mapping the volume number separately or something and displaying the volume as non-hyperlinked text, although they are on a much different Blacklight version than we are: https://searchworks.stanford.edu/view/1595452

Let's hold off on deploying this for now

**Comment by seanluyk** [comment]
@ConnorSheremeta @weiweishi Stanford seems to be doing lots of manipulation for linked and unlinked series, for e.g.: https://github.com/sul-dlss/SearchWorks/blob/615435f4dd12be7c2cd5defe947c720cb7475440/app/models/marc_fields/linked_series.rb

Seems like we're getting into enhancement territory

**Comment by pgwillia** [comment]
There are a couple of hacks we could look at if this is desired fast.  

1. Given a solr response document you can `marc = document.to_marc` then `marc["245"]["a"]` might be the title or marc['490']['a'] might be the series title.
https://github.com/ualbertalib/discovery/blob/8196db76fdf73513404fe3495dc4cbd6ff7c31e1/app/views/catalog/_show_default.html.erb#L14-L17

2. We manipulate the subjects after the fact to reason about parts of them.  Might be able to do something similar with the series. (Note that there is a lot of logic in the view which probably should live in a helper so that it can be tested and not clutter the view)  See https://github.com/ualbertalib/discovery/blob/8196db76fdf73513404fe3495dc4cbd6ff7c31e1/app/controllers/catalog_controller.rb#L41-L46 and https://github.com/ualbertalib/discovery/blob/8196db76fdf73513404fe3495dc4cbd6ff7c31e1/app/views/catalog/show.html.erb#L33-L49

Maybe it's worth taking a step back and think about the big picture of how we could refactor this feature into a Jupiter/JupiterCore like app.

**Comment by seanluyk** [comment]
@ConnorSheremeta I completely agree with your point about taking a step back and looking at the bigger picture, especially if this is more trouble than it's worth. Let's chat about options at our next grooming meeting
